CardsCashCasino/Manager/TexasHoldEmManager.cs

```diff
@@ -470,6 +475,50 @@ public void Update()
 
             int totalPotValue = _potManager.Pots.Sum(pot => pot.Total); // Calculate total pot value
             _potUI.UpdatePot(totalPotValue);
+            
+            // Update the position of the turn indicator depending on who's turn it is
+            UpdateTurnIndicatorPosition();
+        }
+
+        /// <summary>
+        /// Called in the update loop. Updates the position of the turn indicator depending on who's turn it is.
+        /// </summary>
+        private void UpdateTurnIndicatorPosition()
```

Comment:
these seems more complicated than necessary. why not just create a turn indicator option for each player, and then just enable and disable it based on what player is playing?